### ISSUE TITLE
*****PENDING CONFIRMATION SOLVED***** Adios Iota !

### ISSUE THREAD

**Issue by RonRizzo** [iotaledger/wallet#529]
Guys, 
Pending confirmation problem solved!  Here is what I did to get  both of my  transactions confirmed. One was 2 day’s, the other was for 1 day 14 hours.  First and foremost, be persistent and do not relent:

(1)	Change the weight to “15”
(2)	 Click on “History”
(3)	Click on the top pending “bundle” transaction
(4)	REBROADCAST (make sure this step is “completed”)
(5)	REATTACH (make sure this step is “completed”)
(6)	Change NODES and repeat steps 1-5
(7)	Do 3 to 5 times daily or more. 

Note: If the  rebroadcast is not completed with 30 seconds- change nodes! 

 I dumped this garbage coin @ $1.15 and made a nice profit😊 Good Luck!
![solved](https://user-images.githubusercontent.com/33988824/33300067-4608568c-d3a3-11e7-9758-172e9ef014e4.PNG)

**Comment by thivaz** [comment]
how to REATTACH?

**Comment by RonRizzo** [comment]
Good for you!   I've already purchased new potions in a legit company  that
has had  have 50% plus gains since my purchase and is climbing! I have
absolutely no regrets. I'm just glad I was able to get my "confirmations"
and dump this loser coin- They are manipulating the market for sure and it
wont be long before this house of cards comes crashing down- so buy some
more why don't you.

On Tue, Nov 28, 2017 at 8:26 AM, Thomas A. Pepperz <notifications@github.com
> wrote:

> Thanks for dumping at $1.15 it’s at 1.25 and I’m holding on til it’s vakues
> at $125.25. Period.
>
